src/main/idea/spotify-checkstyle-idea.xml

```diff
@@ -41,13 +55,18 @@
     <option name="BINARY_OPERATION_SIGN_ON_NEXT_LINE" value="true" />
     <option name="TERNARY_OPERATION_WRAP" value="1" />
     <option name="TERNARY_OPERATION_SIGNS_ON_NEXT_LINE" value="true" />
+    <option name="KEEP_SIMPLE_METHODS_IN_ONE_LINE" value="true" />
```

Comment:
It has to do with the code formatting tool: if the method didn't have line breaks to start, the formatter won't add them.